lib/v2/magazinelib/radar.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    'magazinelib.com': {
+        _name: 'Magazine',
+        '.': [
+            {
+                title: 'Latest Magazine',
+                docs: 'https://docs.rsshub.app/shopping.html#arcteryx',
+                source: ['/', '/?s=*'],
```

Comment:
```suggestion
                source: ['/'],
```
source cannot match search parameters